packages/esm-ward-app/src/ward-patient-card/ward-patient-card.component.tsx

```diff
@@ -62,16 +68,6 @@ const WardPatientCard: WardPatientCard = (wardPatient) => {
         ))}
         <ExtensionSlot name={footerExtensionSlotName} state={wardPatient} />
       </div>
-      <button
```

Comment:
Previously, the patient card was made clickable by adding this button to overlay on top of the elements of the card. I'm not sure why it was implemented that way, but I had to change it in order for the ToggleTips to receive click events.

cc @brandones

Reply:
It is [widely and strongly recommended against](https://theadminbar.com/accessibility-weekly/divs-are-not-buttons/) adding onClick to a div.

There are myriad articles about how to create a "clickable card" component, which is simply kind of a hard problem, but I really like [this one](https://css-tricks.com/block-links-the-search-for-a-perfect-solution/). I implemented [Method 3](https://css-tricks.com/block-links-the-search-for-a-perfect-solution/#aa-method-3-the-good-ol-before-pseudo-element) from it, which agrees with the recommendations of the widely-cited [Card guidelines](https://inclusive-components.design/cards/) from Inclusive Components.

Rather than using JS as recommended by Method 4 of that article, I would recommend just using the current approach and increasing the z-index of any links you need to be clickable on top of the card. The only reason Method 4 would be preferable would be to make the text selectable, and I don't think that's relevant to this interface.

As an aside, did this ToggleTip thing get designed? It makes the interface look a lot busier, appears to have the info button at the same level of visual hierarchy as the obs information itself, and seems like it would be easy to click the wrong thing. I would suggest making the tooltip appear on hover anywhere over the tag element (with a 500ms delay). You would only need JS to do touch handling on mobile, something like touch-and-hold shows the tooltip while tap clicks the card. You would still need the z-index fix.

Alternatively, since it seems like this is supposed to solve a problem not actually faced by users (but rather by devs & admins) you could add a boolean config option that makes all tags show some metadata. Then you would be able to debug just by switching it in the implementer tools.

Reply:
Ok, makes sense. I'll implement it using z-index. 

> As an aside, did this ToggleTip thing get designed?

No, I did this before I got a [reply from UX designers](https://openmrs.slack.com/archives/C06P2GVUGEP/p1725656280125159). I'll make the whole tag clickable for mobile and hoverable on desktop.

> Alternatively, since it seems like this is supposed to solve a problem not actually faced by users (but rather by devs & admins)

While it's mostly for debugging, I think it's also important to show this info so that mistakes in implementation / configuration are as apparent as possible to the end users.